README.md

```diff
@@ -1 +1,114 @@
-# Hello-world
\ No newline at end of file
+<a name="readme-top"></a>
+
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+  - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+- [👥 Authors](#authors)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
```

Comment:
- [ ] As requested by the previous reviewer, it is important that all main headers from the [README template](https://github.com/microverseinc/readme-template) are present in our repo to ensure that we have a well-descriptive readme file. 👍. The only optional section is the `FAQ`.

- [ ] Kindly check all the main headers of the README template below 👇 

![image](https://user-images.githubusercontent.com/78251927/234923748-f1e2d9fc-d17a-4a5b-b1a7-525cc0366086.png)

- [ ] Your readme 👇 

![image](https://user-images.githubusercontent.com/78251927/234925816-d18cdb95-4919-40c9-887e-728dccf9574e.png)

Kindly add the section **Future Features**, including 1-3 potential features to be added. 👍 🛠